src/containers/Profiles/EditProfile.tsx

```diff
@@ -14,12 +16,21 @@ import Button from '#src/components/Button/Button';
 import { addQueryParam } from '#src/utils/location';
 import FormFeedback from '#src/components/FormFeedback/FormFeedback';
 import { getProfileDetails, updateProfile } from '#src/stores/AccountController';
+import { useListProfiles } from '#src/hooks/useProfiles';
 
-const EditProfile = () => {
-  const { id } = useParams();
+type EditProfileProps = {
+  contained?: boolean;
+};
+
+const EditProfile = ({ contained = false }: EditProfileProps) => {
+  const params = useParams();
+  const { id } = params;
   const location = useLocation();
   const navigate = useNavigate();
   const [fullName, setFullName] = useState<string>('');
+  const { t } = useTranslation('user');
+
+  const listProfiles = useListProfiles();
```

Comment:
The `useListProfiles` uses stores and APIs. Do you think we can extract this to the parent container?

Reply:
Nevermind!